templates/hooks/key/sdk_read_many_post_set_output.go.tpl

```diff
@@ -0,0 +1,3 @@
+if err := rm.setResourceAdditionalFields(ctx, ko); err != nil {
+    return nil, err
+}
```

Comment:
To inject templates hooks into the code you'll need to instruct the code-generator to do that, by editing the `generator.yaml`, using something like https://github.com/aws-controllers-k8s/iam-controller/blob/main/generator.yaml#L121-L128.
Here is the list of possible hooks to inject: https://github.com/aws-controllers-k8s/code-generator/blob/160b839fe09dd7e1f321e094604ffc3b6ae2a285/pkg/generate/ack/hook.go#L28-L71
In this case you need one for the `sdk_create_post_set_output` and one for `sdk_read_many_post_set_output`